go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
```

Comment:
This should not be included, please remove from the commit.

#1209 addresses the removing of the dependency

Reply:
Sure. Will do.